scripts/zones/East_Ronfaure_[S]/mobs/Battrap.lua

```diff
@@ -0,0 +1,25 @@
+-----------------------------------
+-- Area: East Ronfaure [S]
+--  MOB: Battrap
+-- Note:  Place Holder for Goblintrap NM
+-- !pos 156 0 -438 81
+-----------------------------------
+
+require("scripts/zones/East_Ronfaure_[S]/MobIDs");
+
```

Comment:
should also require("scripts/globals/mobs");
this defines the phOnDespawn() function.